server/routes/apply.ts

```diff
@@ -23,6 +23,9 @@ export default function applyRoutes(controllers: Controllers, router: Router, se
   put(paths.applications.appendToList.pattern, applicationsController.appendToList(), {
     auditEvent: 'UPDATE_APPLICATION_LIST',
   })
+  get(paths.applications.removeFromList.pattern, applicationsController.removeFromList(), {
```

Comment:
should this not be a PUT?

Reply:
yes I think PUT or PATCH is more appropriate for this request which changes a list of items...